gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -708,6 +708,11 @@
   public static final String DEFAULT_METRICS_REPORTING_FILE_ENABLED = Boolean.toString(false);
   public static final String METRICS_LOG_DIR_KEY = METRICS_CONFIGURATIONS_PREFIX + "log.dir";
   public static final String METRICS_FILE_SUFFIX = METRICS_CONFIGURATIONS_PREFIX + "reporting.file.suffix";
+  public static final String MR_TYPE_KEY = METRICS_CONFIGURATIONS_PREFIX + "mr.type";
```

Comment:
Since these configuration keys are dynamically set and not user provided, its better to define them inside the MRJobLauncher class.